SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -460,6 +460,16 @@ int64 USpatialActorChannel::ReplicateActor()
 	}
 
 	check(Actor);
+
+	// If the entity already exists, make sure we have authority before we replicate in a multi-worker deployment, because we pretend to have local authority
+	if (!bCreatingNewEntity
+		&& NetDriver->LoadBalanceStrategy != nullptr
+		&& !NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::POSITION_COMPONENT_ID)
+		&& !NetDriver->LoadBalanceStrategy->ShouldHaveAuthority(*Actor))
+	{
+		return 0;
+	}
```

Comment:
What issues does this escape? I can't wrap my head around it. Also, is there a reason for avoiding using a HasAuthority check instead?

Reply:
ULTRA NIT (avoid at your leisure): I think we're trying to end comments in full stops now (although it seems like a lot of people didn't in the past 😄 )

Reply:
This was something I pulled from Tim when I was debugging the fact that SpatialDebugger wasn't showing up. I'll test again without it, because it may be that it's not needed anymore.

Reply:
I tested and this isn't something that's needed post-singleton fix, and maybe it wasn't ever. I'm not sure what it fixed for Tim.